tutorials/metrics/metrics_file_regex_with_linenum.yml

```diff
@@ -0,0 +1,27 @@
+buildspecs:
+  metric_file_regex:
+    executor: generic.local.bash
+    type: script
+    description: capture result metric from file path
+    run: echo -e "HPCG result is INVALID with a GFLOP/s rating of=28.1215\n
```

Comment:
please change this also to using `run: |`

Reply:
👍